NashTeamAlpha/BangazonWeb
# Ddw submit product

## Pull Request Description

Include the applicable information from the list below here 

Fixed DBInitializer file so that the database seeds correctly
Got form to submit data to the database and to the products page
## Steps to Test

Include the applicable information from the list below here
1. Switch to branch `ddw-submit-products`
2. Run `dotnet ef database update`
3. Make sure there are no errors in the code or on run 
## Link to Feature Ticket
#47 #97
